docs/modules/vertiq_8108_150.rst

```diff
@@ -1,13 +1,13 @@
 ********************************
-Vertiq 8108 150KV | Speed Module
+Vertiq 81-08 | Speed Module
 ********************************
 
-.. sidebar:: Vertiq 8108 150KV
+.. sidebar:: Vertiq 81-08
 
     .. image:: ../_static/Vertiq_8108_front.jpg
-        :alt: Vertiq 8108 150KV
+        :alt: Vertiq 81-08 150KV
 
-The Vertiq 8108 150Kv is built for high performance commercial UAV and robotic applications. Its default firmware is the **Speed Firmware**.
+The Vertiq 81-08 150Kv is built for high performance commercial UAV and robotic applications. Its default firmware is the **Speed Firmware**.
```

Comment:
Should we lose the 150 here as well?

Reply:
Asked Jon about this back on december 15, and never got a response. I'll poke

Reply:
Jon is "indifferent" to this as well. He agrees with our future plan to move towards module family pages, but is ok with this for now.